Release 0.23.0-rc1 failed, 145 errors found in scala-maven-plugin:3.2.2:doc

```
$ ./scripts/release/release.sh 0.23.0-rc1 0.23.0-SNAPSHOT
...
[INFO] ------------------------------------------------------------------------
[INFO] Building ADAM_2.11: Core 0.23.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] >>> scala-maven-plugin:3.2.2:doc (default-cli) > generate-sources @ adam-core-spark2_2.11 >>>
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-codegen-spark2_2.11/0.23.0-SNAPSHOT/maven-metadata.xml
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-codegen-spark2_2.11/0.23.0-SNAPSHOT/maven-metadata.xml (1.2 kB at 2.2 kB/s)
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-codegen-spark2_2.11/0.23.0-SNAPSHOT/adam-codegen-spark2_2.11-0.23.0-20171228.152237-370.jar
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-codegen-spark2_2.11/0.23.0-SNAPSHOT/adam-codegen-spark2_2.11-0.23.0-20171228.152237-370.jar (33 kB at 217 kB/s)
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-versions) @ adam-core-spark2_2.11 ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-maven) @ adam-core-spark2_2.11 ---
[INFO] 
[INFO] --- build-helper-maven-plugin:1.12:add-source (add-source) @ adam-core-spark2_2.11 ---
[INFO] Source directory: /Users/heuermh/working/adam/adam-core/src/main/scala added.
[INFO] Source directory: /Users/heuermh/working/adam/adam-core/target/generated-sources/src/main/scala added.
[INFO] 
[INFO] --- exec-maven-plugin:1.5.0:java (generate-scala-products) @ adam-core-spark2_2.11 ---
[INFO] 
[INFO] --- exec-maven-plugin:1.5.0:java (generate-scala-projection-fields) @ adam-core-spark2_2.11 ---
[INFO] 
[INFO] <<< scala-maven-plugin:3.2.2:doc (default-cli) < generate-sources @ adam-core-spark2_2.11 <<<
[INFO] 
[INFO] 
[INFO] --- scala-maven-plugin:3.2.2:doc (default-cli) @ adam-core-spark2_2.11 ---
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-codegen-spark2_2.11/0.23.0-SNAPSHOT/adam-codegen-spark2_2.11-0.23.0-20171228.152237-370.pom
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-codegen-spark2_2.11/0.23.0-SNAPSHOT/adam-codegen-spark2_2.11-0.23.0-20171228.152237-370.pom (3.8 kB at 34 kB/s)
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/converters/FastaConverter.scala:222: warning: discarding unmoored doc comment
    /**
    ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:746: warning: Octal escape literals are deprecated, use \u0001 instead.
      binaryCodec.writeBytes("BAM\001".getBytes())
                                 ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rich/RichCigar.scala:43: warning: discarding unmoored doc comment
    /**
    ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantRDD.scala:39: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ Variant => VariantProduct }
                           ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantRDD.scala:197: error: not found: type VariantProduct
sealed abstract class VariantRDD extends AvroGenomicRDD[Variant, VariantProduct, VariantRDD] {
                                                                 ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:76: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{
                           ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:926: error: not found: type GenotypeProduct
    ds: Dataset[GenotypeProduct]): GenotypeRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:907: error: not found: type AlignmentRecordProduct
    ds: Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:888: error: not found: type FragmentProduct
    ds: Dataset[FragmentProduct]): FragmentRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:872: error: not found: type FeatureProduct
    ds: Dataset[FeatureProduct]): FeatureRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:848: error: not found: type NucleotideContigFragmentProduct
    ds: Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:825: error: not found: type VariantProduct
    ds: Dataset[VariantProduct]): VariantRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:807: error: not found: type AlignmentRecordProduct
    ds: Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:788: error: not found: type FragmentProduct
    ds: Dataset[FragmentProduct]): FragmentRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:772: error: not found: type FeatureProduct
    ds: Dataset[FeatureProduct]): FeatureRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:748: error: not found: type NucleotideContigFragmentProduct
    ds: Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:725: error: not found: type VariantProduct
    ds: Dataset[VariantProduct]): VariantRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:707: error: not found: type GenotypeProduct
    ds: Dataset[GenotypeProduct]): GenotypeRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:688: error: not found: type FragmentProduct
    ds: Dataset[FragmentProduct]): FragmentRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:672: error: not found: type FeatureProduct
    ds: Dataset[FeatureProduct]): FeatureRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:648: error: not found: type NucleotideContigFragmentProduct
    ds: Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:558: error: not found: type VariantProduct
    ds: Dataset[VariantProduct]): VariantRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:540: error: not found: type GenotypeProduct
    ds: Dataset[GenotypeProduct]): GenotypeRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:521: error: not found: type AlignmentRecordProduct
    ds: Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:505: error: not found: type FeatureProduct
    ds: Dataset[FeatureProduct]): FeatureRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:481: error: not found: type NucleotideContigFragmentProduct
    ds: Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:458: error: not found: type VariantProduct
    ds: Dataset[VariantProduct]): VariantRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:440: error: not found: type GenotypeProduct
    ds: Dataset[GenotypeProduct]): GenotypeRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:421: error: not found: type AlignmentRecordProduct
    ds: Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:402: error: not found: type FragmentProduct
    ds: Dataset[FragmentProduct]): FragmentRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:374: error: not found: type NucleotideContigFragmentProduct
    ds: Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:351: error: not found: type VariantProduct
    ds: Dataset[VariantProduct]): VariantRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:333: error: not found: type GenotypeProduct
    ds: Dataset[GenotypeProduct]): GenotypeRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:314: error: not found: type AlignmentRecordProduct
    ds: Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:295: error: not found: type FragmentProduct
    ds: Dataset[FragmentProduct]): FragmentRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:279: error: not found: type FeatureProduct
    ds: Dataset[FeatureProduct]): FeatureRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:267: error: not found: type NucleotideContigFragmentProduct
    ds: Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:244: error: not found: type VariantProduct
    ds: Dataset[VariantProduct]): VariantRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:226: error: not found: type GenotypeProduct
    ds: Dataset[GenotypeProduct]): GenotypeRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:207: error: not found: type AlignmentRecordProduct
    ds: Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:188: error: not found: type FragmentProduct
    ds: Dataset[FragmentProduct]): FragmentRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:172: error: not found: type FeatureProduct
    ds: Dataset[FeatureProduct]): FeatureRDD = {
                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:21: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ Feature => FeatureProduct }
                           ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:103: error: not found: type FeatureProduct
  def toSqlFeature: FeatureProduct = {
                    ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:104: error: not found: type FeatureProduct
    new FeatureProduct(featureId = None,
        ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:104: error: not found: value featureId
    new FeatureProduct(featureId = None,
                       ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:105: error: not found: value name
      name = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:106: error: not found: value source
      source = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:107: error: not found: value featureType
      featureType = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:108: error: reassignment to val
      contigName = Some(contigName),
                 ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:109: error: reassignment to val
      start = Some(start),
            ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:110: error: reassignment to val
      end = Some(end),
          ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:111: error: not found: value strand
      strand = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:112: error: not found: value phase
      phase = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:113: error: not found: value frame
      frame = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:114: error: not found: value score
      score = Some(count),
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:115: error: not found: value geneId
      geneId = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:116: error: not found: value transcriptId
      transcriptId = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:117: error: not found: value exonId
      exonId = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:118: error: not found: value aliases
      aliases = Seq.empty,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:119: error: not found: value parentIds
      parentIds = Seq.empty,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:120: error: not found: value target
      target = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:121: error: not found: value gap
      gap = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:122: error: not found: value derivesFrom
      derivesFrom = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:123: error: not found: value notes
      notes = Seq.empty,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:124: error: not found: value dbxrefs
      dbxrefs = Seq.empty,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:125: error: not found: value ontologyTerms
      ontologyTerms = Seq.empty,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:126: error: not found: value circular
      circular = None,
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:127: error: not found: value attributes
      attributes = Map.empty)
      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:50: error: object FeatureField is not a member of package org.bdgenomics.adam.projections
import org.bdgenomics.adam.projections.{
       ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:36: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ NucleotideContigFragment => NucleotideContigFragmentProduct }
                           ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:188: error: not found: type NucleotideContigFragmentProduct
sealed abstract class NucleotideContigFragmentRDD extends AvroGenomicRDD[NucleotideContigFragment, NucleotideContigFragmentProduct, NucleotideContigFragmentRDD] {
                                                                                                   ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:36: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ Feature => FeatureProduct }
                           ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:339: error: not found: type FeatureProduct
sealed abstract class FeatureRDD extends AvroGenomicRDD[Feature, FeatureProduct, FeatureRDD] {
                                                                 ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:277: error: not found: type FeatureProduct
    dataset: Dataset[FeatureProduct],
                     ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:41: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ Fragment => FragmentProduct }
                           ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:259: error: not found: type FragmentProduct
sealed abstract class FragmentRDD extends AvroRecordGroupGenomicRDD[Fragment, FragmentProduct, FragmentRDD] {
                                                                              ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:183: error: not found: type FragmentProduct
    dataset: Dataset[FragmentProduct],
                     ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:60: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ AlignmentRecord => AlignmentRecordProduct }
                           ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:343: error: not found: type AlignmentRecordProduct
sealed abstract class AlignmentRecordRDD extends AvroRecordGroupGenomicRDD[AlignmentRecord, AlignmentRecordProduct, AlignmentRecordRDD] {
                                                                                            ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:242: error: not found: type AlignmentRecordProduct
    dataset: Dataset[AlignmentRecordProduct],
                     ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala:38: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ Genotype => GenotypeProduct }
                           ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala:213: error: not found: type GenotypeProduct
sealed abstract class GenotypeRDD extends MultisampleAvroGenomicRDD[Genotype, GenotypeProduct, GenotypeRDD] {
                                                                              ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala:141: error: not found: type GenotypeProduct
    dataset: Dataset[GenotypeProduct],
                     ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantRDD.scala:135: error: not found: type VariantProduct
    dataset: Dataset[VariantProduct],
                     ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:135: error: not found: type NucleotideContigFragmentProduct
    dataset: Dataset[NucleotideContigFragmentProduct],
                     ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:195: error: not found: type AlignmentRecordProduct
  def apply(ds: Dataset[AlignmentRecordProduct],
                        ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:2386: error: not found: value FeatureField
      val coverageFields = Projection(FeatureField.contigName,
                                      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:2387: error: not found: value FeatureField
        FeatureField.start,
        ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:2388: error: not found: value FeatureField
        FeatureField.end,
        ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:2389: error: not found: value FeatureField
        FeatureField.score)
        ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:121: error: not found: type FeatureProduct
  def apply(ds: Dataset[FeatureProduct],
                        ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:125: error: not found: type NucleotideContigFragmentProduct
    sqlContext.read.parquet(parquetFilename).as[NucleotideContigFragmentProduct]
                                                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:176: error: not found: type NucleotideContigFragmentProduct
  lazy val dataset: Dataset[NucleotideContigFragmentProduct] = {
                            ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:179: error: not found: value NucleotideContigFragmentProduct
    sqlContext.createDataset(rdd.map(NucleotideContigFragmentProduct.fromAvro))
                                     ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:190: error: not found: type NucleotideContigFragmentProduct
  @transient val uTag: TypeTag[NucleotideContigFragmentProduct] = typeTag[NucleotideContigFragmentProduct]
                               ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:190: error: not found: type NucleotideContigFragmentProduct
  @transient val uTag: TypeTag[NucleotideContigFragmentProduct] = typeTag[NucleotideContigFragmentProduct]
                                                                          ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:244: error: not found: type NucleotideContigFragmentProduct
    tFn: Dataset[NucleotideContigFragmentProduct] => Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                 ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:244: error: not found: type NucleotideContigFragmentProduct
    tFn: Dataset[NucleotideContigFragmentProduct] => Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                                                             ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/CoverageRDD.scala:116: error: ambiguous implicit values:
 both method newIntEncoder in class SQLImplicits of type => org.apache.spark.sql.Encoder[Int]
 and method newLongEncoder in class SQLImplicits of type => org.apache.spark.sql.Encoder[Long]
 match expected type org.apache.spark.sql.Encoder[U]
    DatasetBoundFeatureRDD(dataset.map(_.toSqlFeature), sequences)
                                      ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:264: error: not found: type FeatureProduct
    sqlContext.read.parquet(parquetFilename).as[FeatureProduct]
                                                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:363: error: not found: type FeatureProduct
    tFn: Dataset[FeatureProduct] => Dataset[FeatureProduct]): FeatureRDD = {
                 ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:363: error: not found: type FeatureProduct
    tFn: Dataset[FeatureProduct] => Dataset[FeatureProduct]): FeatureRDD = {
                                            ^
warning: there were 6 feature warnings; re-run with -feature for details
model contains 216 documentable templates
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/util/AttributeUtils.scala:70: warning: Could not find any member to link for "IllegalArgumentException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/projections/Projection.scala:45: warning: Could not find any member to link for "IllegalArgumentException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:338: warning: Could not find any member to link for "UnsupportedOperationException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:1276: warning: Could not find any member to link for "FileNotFoundException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:1305: warning: Could not find any member to link for "FileNotFoundException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:1325: warning: Could not find any member to link for "FileNotFoundException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicPartitioners.scala:73: warning: Could not find any member to link for "IllegalArgumentException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicPartitioners.scala:176: warning: Could not find any member to link for "IllegalArgumentException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala:428: warning: Could not find any member to link for "IllegalArgumentException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:2740: warning: Could not find any member to link for "IllegalArgumentException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala:383: warning: Could not find any member to link for "IllegalArgumentException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala:440: warning: Variable x undefined in comment for method pipe in trait GenomicRDD
   * Files are substituted in to the command with a $x syntax. E.g., to invoke
                                                     ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala:441: warning: Variable 0 undefined in comment for method pipe in trait GenomicRDD
   * a command that uses the first file from the files Seq, use $0. To access
                                                                 ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala:442: warning: Variable root undefined in comment for method pipe in trait GenomicRDD
   * the path to the directory where the files are copied, use $root.
                                                                ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/RecordGroupDictionary.scala:47: warning: Could not find any member to link for "IllegalArgumentError".
/**
^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Alphabet.scala:44: warning: Could not find any member to link for "IllegalArgumentException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala:230: warning: Could not find any member to link for "IllegalArgumentException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala:214: warning: Could not find any member to link for "IllegalArgumentException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala:398: warning: Could not find any member to link for "IllegalArgumentException".
  /**
  ^
/Users/heuermh/working/adam/adam-core/src/main/scala/org/bdgenomics/adam/models/ReferenceRegion.scala:308: warning: Could not find any member to link for "IllegalArgumentException".
  /**
  ^
24 warnings found
145 errors found
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] ADAM_2.11 .......................................... SUCCESS [  0.711 s]
[INFO] ADAM_2.11: Avro-to-Dataset codegen utils ........... SUCCESS [  3.803 s]
[INFO] ADAM_2.11: Core .................................... FAILURE [ 30.808 s]
[INFO] ADAM_2.11: APIs for Java ........................... SKIPPED
[INFO] ADAM_2.11: CLI ..................................... SKIPPED
[INFO] ADAM_2.11: Assembly ................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 35.626 s
[INFO] Finished at: 2017-12-28T10:25:15-06:00
[INFO] Final Memory: 35M/658M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal net.alchim31.maven:scala-maven-plugin:3.2.2:doc (default-cli) on project adam-core-spark2_2.11: wrap: org.apache.maven.reporting.MavenReportException: wrap: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :adam-core-spark2_2.11
Switched to branch 'master'
Your branch is up to date with 'origin/master'.
[INFO] Scanning for projects...
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Build Order:
[INFO] 
[INFO] ADAM_2.10
[INFO] ADAM_2.10: Avro-to-Dataset codegen utils
[INFO] ADAM_2.10: Core
[INFO] ADAM_2.10: APIs for Java
[INFO] ADAM_2.10: CLI
[INFO] ADAM_2.10: Assembly
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Building ADAM_2.10 0.23.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- versions-maven-plugin:2.2:set (default-cli) @ adam-parent_2.10 ---
[INFO] Searching for local aggregator root...
[INFO] Local aggregation root: /Users/heuermh/working/adam
[INFO] Processing change of org.bdgenomics.adam:adam-parent_2.10:0.23.0-SNAPSHOT -> 0.23.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] ADAM_2.10 .......................................... SUCCESS [  0.732 s]
[INFO] ADAM_2.10: Avro-to-Dataset codegen utils ........... SKIPPED
[INFO] ADAM_2.10: Core .................................... SKIPPED
[INFO] ADAM_2.10: APIs for Java ........................... SKIPPED
[INFO] ADAM_2.10: CLI ..................................... SKIPPED
[INFO] ADAM_2.10: Assembly ................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 1.351 s
[INFO] Finished at: 2017-12-28T10:25:18-06:00
[INFO] Final Memory: 20M/309M
[INFO] ------------------------------------------------------------------------
On branch master
```

If I run `mvn scala:doc` from a fresh clone, it also encounters doc errors, but does not fail the build.

```
$ git clone https://github.com/bigdatagenomics/adam.git tmp-adam
$ cd tmp-adam/
$ mvn scala:doc
...
[INFO] 
[INFO] --- scala-maven-plugin:3.2.2:doc (default-cli) @ adam-core_2.10 ---
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-codegen_2.10/0.23.0-SNAPSHOT/adam-codegen_2.10-0.23.0-20171228.151829-427.pom
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-codegen_2.10/0.23.0-SNAPSHOT/adam-codegen_2.10-0.23.0-20171228.151829-427.pom (3.8 kB at 32 kB/s)
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantRDD.scala:39: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ Variant => VariantProduct }
                           ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantRDD.scala:197: error: not found: type VariantProduct
sealed abstract class VariantRDD extends AvroGenomicRDD[Variant, VariantProduct, VariantRDD] {
                                                                 ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:76: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{
                           ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:926: error: not found: type GenotypeProduct
    ds: Dataset[GenotypeProduct]): GenotypeRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:907: error: not found: type AlignmentRecordProduct
    ds: Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:888: error: not found: type FragmentProduct
    ds: Dataset[FragmentProduct]): FragmentRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:872: error: not found: type FeatureProduct
    ds: Dataset[FeatureProduct]): FeatureRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:848: error: not found: type NucleotideContigFragmentProduct
    ds: Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:825: error: not found: type VariantProduct
    ds: Dataset[VariantProduct]): VariantRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:807: error: not found: type AlignmentRecordProduct
    ds: Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:788: error: not found: type FragmentProduct
    ds: Dataset[FragmentProduct]): FragmentRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:772: error: not found: type FeatureProduct
    ds: Dataset[FeatureProduct]): FeatureRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:748: error: not found: type NucleotideContigFragmentProduct
    ds: Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:725: error: not found: type VariantProduct
    ds: Dataset[VariantProduct]): VariantRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:707: error: not found: type GenotypeProduct
    ds: Dataset[GenotypeProduct]): GenotypeRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:688: error: not found: type FragmentProduct
    ds: Dataset[FragmentProduct]): FragmentRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:672: error: not found: type FeatureProduct
    ds: Dataset[FeatureProduct]): FeatureRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:648: error: not found: type NucleotideContigFragmentProduct
    ds: Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:558: error: not found: type VariantProduct
    ds: Dataset[VariantProduct]): VariantRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:540: error: not found: type GenotypeProduct
    ds: Dataset[GenotypeProduct]): GenotypeRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:521: error: not found: type AlignmentRecordProduct
    ds: Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:505: error: not found: type FeatureProduct
    ds: Dataset[FeatureProduct]): FeatureRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:481: error: not found: type NucleotideContigFragmentProduct
    ds: Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:458: error: not found: type VariantProduct
    ds: Dataset[VariantProduct]): VariantRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:440: error: not found: type GenotypeProduct
    ds: Dataset[GenotypeProduct]): GenotypeRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:421: error: not found: type AlignmentRecordProduct
    ds: Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:402: error: not found: type FragmentProduct
    ds: Dataset[FragmentProduct]): FragmentRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:374: error: not found: type NucleotideContigFragmentProduct
    ds: Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:351: error: not found: type VariantProduct
    ds: Dataset[VariantProduct]): VariantRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:333: error: not found: type GenotypeProduct
    ds: Dataset[GenotypeProduct]): GenotypeRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:314: error: not found: type AlignmentRecordProduct
    ds: Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:295: error: not found: type FragmentProduct
    ds: Dataset[FragmentProduct]): FragmentRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:279: error: not found: type FeatureProduct
    ds: Dataset[FeatureProduct]): FeatureRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:267: error: not found: type NucleotideContigFragmentProduct
    ds: Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:244: error: not found: type VariantProduct
    ds: Dataset[VariantProduct]): VariantRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:226: error: not found: type GenotypeProduct
    ds: Dataset[GenotypeProduct]): GenotypeRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:207: error: not found: type AlignmentRecordProduct
    ds: Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:188: error: not found: type FragmentProduct
    ds: Dataset[FragmentProduct]): FragmentRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:172: error: not found: type FeatureProduct
    ds: Dataset[FeatureProduct]): FeatureRDD = {
                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:21: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ Feature => FeatureProduct }
                           ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:103: error: not found: type FeatureProduct
  def toSqlFeature: FeatureProduct = {
                    ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/models/Coverage.scala:104: error: not found: type FeatureProduct
    new FeatureProduct(featureId = None,
        ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:50: error: object FeatureField is not a member of package org.bdgenomics.adam.projections
import org.bdgenomics.adam.projections.{
       ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:36: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ NucleotideContigFragment => NucleotideContigFragmentProduct }
                           ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:188: error: not found: type NucleotideContigFragmentProduct
sealed abstract class NucleotideContigFragmentRDD extends AvroGenomicRDD[NucleotideContigFragment, NucleotideContigFragmentProduct, NucleotideContigFragmentRDD] {
                                                                                                   ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:36: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ Feature => FeatureProduct }
                           ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:339: error: not found: type FeatureProduct
sealed abstract class FeatureRDD extends AvroGenomicRDD[Feature, FeatureProduct, FeatureRDD] {
                                                                 ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:277: error: not found: type FeatureProduct
    dataset: Dataset[FeatureProduct],
                     ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:41: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ Fragment => FragmentProduct }
                           ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:259: error: not found: type FragmentProduct
sealed abstract class FragmentRDD extends AvroRecordGroupGenomicRDD[Fragment, FragmentProduct, FragmentRDD] {
                                                                              ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:183: error: not found: type FragmentProduct
    dataset: Dataset[FragmentProduct],
                     ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:60: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ AlignmentRecord => AlignmentRecordProduct }
                           ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:343: error: not found: type AlignmentRecordProduct
sealed abstract class AlignmentRecordRDD extends AvroRecordGroupGenomicRDD[AlignmentRecord, AlignmentRecordProduct, AlignmentRecordRDD] {
                                                                                            ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:242: error: not found: type AlignmentRecordProduct
    dataset: Dataset[AlignmentRecordProduct],
                     ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala:38: error: object sql is not a member of package org.bdgenomics.adam
import org.bdgenomics.adam.sql.{ Genotype => GenotypeProduct }
                           ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala:213: error: not found: type GenotypeProduct
sealed abstract class GenotypeRDD extends MultisampleAvroGenomicRDD[Genotype, GenotypeProduct, GenotypeRDD] {
                                                                              ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala:141: error: not found: type GenotypeProduct
    dataset: Dataset[GenotypeProduct],
                     ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/VariantRDD.scala:135: error: not found: type VariantProduct
    dataset: Dataset[VariantProduct],
                     ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:135: error: not found: type NucleotideContigFragmentProduct
    dataset: Dataset[NucleotideContigFragmentProduct],
                     ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:195: error: not found: type AlignmentRecordProduct
  def apply(ds: Dataset[AlignmentRecordProduct],
                        ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/ADAMContext.scala:2386: error: not found: value FeatureField
      val coverageFields = Projection(FeatureField.contigName,
                                      ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:121: error: not found: type FeatureProduct
  def apply(ds: Dataset[FeatureProduct],
                        ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:125: error: not found: type NucleotideContigFragmentProduct
    sqlContext.read.parquet(parquetFilename).as[NucleotideContigFragmentProduct]
                                                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:125: error: ambiguous implicit values:
 both method newProductEncoder in class SQLImplicits of type [T <: Product](implicit evidence$1: reflect.runtime.universe.TypeTag[T])org.apache.spark.sql.Encoder[T]
 and method newIntEncoder in class SQLImplicits of type => org.apache.spark.sql.Encoder[Int]
 match expected type org.apache.spark.sql.Encoder[<error>]
    sqlContext.read.parquet(parquetFilename).as[NucleotideContigFragmentProduct]
                                               ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:176: error: not found: type NucleotideContigFragmentProduct
  lazy val dataset: Dataset[NucleotideContigFragmentProduct] = {
                            ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:179: error: not found: value NucleotideContigFragmentProduct
    sqlContext.createDataset(rdd.map(NucleotideContigFragmentProduct.fromAvro))
                                     ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:190: error: not found: type NucleotideContigFragmentProduct
  @transient val uTag: TypeTag[NucleotideContigFragmentProduct] = typeTag[NucleotideContigFragmentProduct]
                               ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:190: error: not found: type NucleotideContigFragmentProduct
  @transient val uTag: TypeTag[NucleotideContigFragmentProduct] = typeTag[NucleotideContigFragmentProduct]
                                                                          ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:244: error: not found: type NucleotideContigFragmentProduct
    tFn: Dataset[NucleotideContigFragmentProduct] => Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                 ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/contig/NucleotideContigFragmentRDD.scala:244: error: not found: type NucleotideContigFragmentProduct
    tFn: Dataset[NucleotideContigFragmentProduct] => Dataset[NucleotideContigFragmentProduct]): NucleotideContigFragmentRDD = {
                                                             ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/CoverageRDD.scala:116: error: ambiguous implicit values:
 both method newProductEncoder in class SQLImplicits of type [T <: Product](implicit evidence$1: reflect.runtime.universe.TypeTag[T])org.apache.spark.sql.Encoder[T]
 and method newIntEncoder in class SQLImplicits of type => org.apache.spark.sql.Encoder[Int]
 match expected type org.apache.spark.sql.Encoder[U]
    DatasetBoundFeatureRDD(dataset.map(_.toSqlFeature), sequences)
                                      ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:264: error: not found: type FeatureProduct
    sqlContext.read.parquet(parquetFilename).as[FeatureProduct]
                                                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:264: error: ambiguous implicit values:
 both method newProductEncoder in class SQLImplicits of type [T <: Product](implicit evidence$1: reflect.runtime.universe.TypeTag[T])org.apache.spark.sql.Encoder[T]
 and method newIntEncoder in class SQLImplicits of type => org.apache.spark.sql.Encoder[Int]
 match expected type org.apache.spark.sql.Encoder[<error>]
    sqlContext.read.parquet(parquetFilename).as[FeatureProduct]
                                               ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:280: error: ambiguous implicit values:
 both value SymbolTag in trait Symbols of type => scala.reflect.ClassTag[reflect.runtime.universe.Symbol]
 and value TypeSymbolTag in trait Symbols of type => scala.reflect.ClassTag[reflect.runtime.universe.TypeSymbol]
 match expected type scala.reflect.ClassTag[U]
  lazy val rdd = dataset.rdd.map(_.toAvro)
                                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:363: error: not found: type FeatureProduct
    tFn: Dataset[FeatureProduct] => Dataset[FeatureProduct]): FeatureRDD = {
                 ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:363: error: not found: type FeatureProduct
    tFn: Dataset[FeatureProduct] => Dataset[FeatureProduct]): FeatureRDD = {
                                            ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:298: error: not found: type FeatureProduct
    tFn: Dataset[FeatureProduct] => Dataset[FeatureProduct]): FeatureRDD = {
                 ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:298: error: not found: type FeatureProduct
    tFn: Dataset[FeatureProduct] => Dataset[FeatureProduct]): FeatureRDD = {
                                            ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:323: error: not found: type FeatureProduct
  lazy val dataset: Dataset[FeatureProduct] = {
                            ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:326: error: not found: value FeatureProduct
    sqlContext.createDataset(rdd.map(FeatureProduct.fromAvro))
                                     ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:341: error: not found: type FeatureProduct
  @transient val uTag: TypeTag[FeatureProduct] = typeTag[FeatureProduct]
                               ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/feature/FeatureRDD.scala:341: error: not found: type FeatureProduct
  @transient val uTag: TypeTag[FeatureProduct] = typeTag[FeatureProduct]
                                                         ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:139: error: not found: type FragmentProduct
  def apply(ds: Dataset[FragmentProduct],
                        ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:163: error: not found: type FragmentProduct
    sqlContext.read.parquet(parquetFilename).as[FragmentProduct]
                                                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:163: error: ambiguous implicit values:
 both method newProductEncoder in class SQLImplicits of type [T <: Product](implicit evidence$1: reflect.runtime.universe.TypeTag[T])org.apache.spark.sql.Encoder[T]
 and method newIntEncoder in class SQLImplicits of type => org.apache.spark.sql.Encoder[Int]
 match expected type org.apache.spark.sql.Encoder[<error>]
    sqlContext.read.parquet(parquetFilename).as[FragmentProduct]
                                               ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:188: error: ambiguous implicit values:
 both value SymbolTag in trait Symbols of type => scala.reflect.ClassTag[reflect.runtime.universe.Symbol]
 and value TypeSymbolTag in trait Symbols of type => scala.reflect.ClassTag[reflect.runtime.universe.TypeSymbol]
 match expected type scala.reflect.ClassTag[U]
  lazy val rdd = dataset.rdd.map(_.toAvro)
                                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:301: error: not found: type FragmentProduct
    tFn: Dataset[FragmentProduct] => Dataset[FragmentProduct]): FragmentRDD = {
                 ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:301: error: not found: type FragmentProduct
    tFn: Dataset[FragmentProduct] => Dataset[FragmentProduct]): FragmentRDD = {
                                             ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:207: error: not found: type FragmentProduct
    tFn: Dataset[FragmentProduct] => Dataset[FragmentProduct]): FragmentRDD = {
                 ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:207: error: not found: type FragmentProduct
    tFn: Dataset[FragmentProduct] => Dataset[FragmentProduct]): FragmentRDD = {
                                             ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:237: error: not found: type FragmentProduct
  lazy val dataset: Dataset[FragmentProduct] = {
                            ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:240: error: not found: value FragmentProduct
    sqlContext.createDataset(rdd.map(FragmentProduct.fromAvro))
                                     ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:261: error: not found: type FragmentProduct
  @transient val uTag: TypeTag[FragmentProduct] = typeTag[FragmentProduct]
                               ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/fragment/FragmentRDD.scala:261: error: not found: type FragmentProduct
  @transient val uTag: TypeTag[FragmentProduct] = typeTag[FragmentProduct]
                                                          ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:222: error: not found: type AlignmentRecordProduct
    sqlContext.read.parquet(parquetFilename).as[AlignmentRecordProduct]
                                                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:222: error: ambiguous implicit values:
 both method newProductEncoder in class SQLImplicits of type [T <: Product](implicit evidence$1: reflect.runtime.universe.TypeTag[T])org.apache.spark.sql.Encoder[T]
 and method newIntEncoder in class SQLImplicits of type => org.apache.spark.sql.Encoder[Int]
 match expected type org.apache.spark.sql.Encoder[<error>]
    sqlContext.read.parquet(parquetFilename).as[AlignmentRecordProduct]
                                               ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:247: error: ambiguous implicit values:
 both value SymbolTag in trait Symbols of type => scala.reflect.ClassTag[reflect.runtime.universe.Symbol]
 and value TypeSymbolTag in trait Symbols of type => scala.reflect.ClassTag[reflect.runtime.universe.TypeSymbol]
 match expected type scala.reflect.ClassTag[U]
  lazy val rdd = dataset.rdd.map(_.toAvro)
                                ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:356: error: not found: type AlignmentRecordProduct
    tFn: Dataset[AlignmentRecordProduct] => Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                 ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:356: error: not found: type AlignmentRecordProduct
    tFn: Dataset[AlignmentRecordProduct] => Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                                                    ^
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDD.scala:266: error: not found: type AlignmentRecordProduct
    tFn: Dataset[AlignmentRecordProduct] => Dataset[AlignmentRecordProduct]): AlignmentRecordRDD = {
                 ^
warning: there were 6 feature warning(s); re-run with -feature for details
model contains 216 documentable templates
/Users/heuermh/working/tmp-adam/adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala:464: warning: Variable x undefined in comment for method pipe in class VariantRDD
  def pipe[X, Y <: GenomicRDD[X, Y], V <: InFormatter[T, U, V]](cmd: String,
      ^
two warnings found
129 errors found
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Building ADAM_2.10: APIs for Java 0.23.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] >>> scala-maven-plugin:3.2.2:doc (default-cli) > generate-sources @ adam-apis_2.10 >>>
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-core_2.10/0.23.0-SNAPSHOT/maven-metadata.xml
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-core_2.10/0.23.0-SNAPSHOT/maven-metadata.xml (1.4 kB at 9.7 kB/s)
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-core_2.10/0.23.0-SNAPSHOT/adam-core_2.10-0.23.0-20171228.151932-444.jar
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-core_2.10/0.23.0-SNAPSHOT/adam-core_2.10-0.23.0-20171228.151932-444-tests.jar
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-core_2.10/0.23.0-SNAPSHOT/adam-core_2.10-0.23.0-20171228.151932-444.jar (3.4 MB at 1.8 MB/s)
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-core_2.10/0.23.0-SNAPSHOT/adam-core_2.10-0.23.0-20171228.151932-444-tests.jar (4.5 MB at 2.2 MB/s)
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-versions) @ adam-apis_2.10 ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-maven) @ adam-apis_2.10 ---
[INFO] 
[INFO] --- build-helper-maven-plugin:1.12:add-source (add-source) @ adam-apis_2.10 ---
[INFO] Source directory: /Users/heuermh/working/tmp-adam/adam-apis/src/main/scala added.
[INFO] 
[INFO] <<< scala-maven-plugin:3.2.2:doc (default-cli) < generate-sources @ adam-apis_2.10 <<<
[INFO] 
[INFO] 
[INFO] --- scala-maven-plugin:3.2.2:doc (default-cli) @ adam-apis_2.10 ---
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-core_2.10/0.23.0-SNAPSHOT/adam-core_2.10-0.23.0-20171228.151932-444.pom
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-core_2.10/0.23.0-SNAPSHOT/adam-core_2.10-0.23.0-20171228.151932-444.pom (12 kB at 90 kB/s)
[WARNING]  Expected all dependencies to require Scala version: 2.10.6
[WARNING]  org.bdgenomics.adam:adam-apis_2.10:0.23.0-SNAPSHOT requires scala version: 2.10.6
[WARNING]  com.twitter:chill_2.10:0.5.0 requires scala version: 2.10.6
[WARNING]  com.typesafe.akka:akka-remote_2.10:2.3.11 requires scala version: 2.10.6
[WARNING]  com.typesafe.akka:akka-actor_2.10:2.3.11 requires scala version: 2.10.6
[WARNING]  com.typesafe.akka:akka-slf4j_2.10:2.3.11 requires scala version: 2.10.6
[WARNING]  org.apache.spark:spark-core_2.10:1.6.3 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-jackson_2.10:3.2.10 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-core_2.10:3.2.10 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-ast_2.10:3.2.10 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-core_2.10:3.2.10 requires scala version: 2.10.0
[WARNING] Multiple versions of scala libraries detected!
[WARNING]  Expected all dependencies to require Scala version: 2.10.6
[WARNING]  org.bdgenomics.adam:adam-apis_2.10:0.23.0-SNAPSHOT requires scala version: 2.10.6
[WARNING]  com.twitter:chill_2.10:0.5.0 requires scala version: 2.10.6
[WARNING]  com.typesafe.akka:akka-remote_2.10:2.3.11 requires scala version: 2.10.6
[WARNING]  com.typesafe.akka:akka-actor_2.10:2.3.11 requires scala version: 2.10.6
[WARNING]  com.typesafe.akka:akka-slf4j_2.10:2.3.11 requires scala version: 2.10.6
[WARNING]  org.apache.spark:spark-core_2.10:1.6.3 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-jackson_2.10:3.2.10 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-core_2.10:3.2.10 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-ast_2.10:3.2.10 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-core_2.10:3.2.10 requires scala version: 2.10.0
[WARNING] Multiple versions of scala libraries detected!
warning: there were 2 feature warning(s); re-run with -feature for details
model contains 124 documentable templates
one warning found
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Building ADAM_2.10: CLI 0.23.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] >>> scala-maven-plugin:3.2.2:doc (default-cli) > generate-sources @ adam-cli_2.10 >>>
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-apis_2.10/0.23.0-SNAPSHOT/maven-metadata.xml
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-apis_2.10/0.23.0-SNAPSHOT/maven-metadata.xml (1.4 kB at 13 kB/s)
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-apis_2.10/0.23.0-SNAPSHOT/adam-apis_2.10-0.23.0-20171228.152022-444.jar
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-apis_2.10/0.23.0-SNAPSHOT/adam-apis_2.10-0.23.0-20171228.152022-444-tests.jar
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-apis_2.10/0.23.0-SNAPSHOT/adam-apis_2.10-0.23.0-20171228.152022-444-tests.jar (25 kB at 171 kB/s)
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-apis_2.10/0.23.0-SNAPSHOT/adam-apis_2.10-0.23.0-20171228.152022-444.jar (182 kB at 860 kB/s)
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-versions) @ adam-cli_2.10 ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-maven) @ adam-cli_2.10 ---
[INFO] 
[INFO] --- git-commit-id-plugin:2.2.1:revision (default) @ adam-cli_2.10 ---
[INFO] 
[INFO] --- templating-maven-plugin:1.0.0:filter-sources (filter-src) @ adam-cli_2.10 ---
[INFO] Coping files with filtering to temporary directory.
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 1 resource
[INFO] Copied 1 files to output directory: /Users/heuermh/working/tmp-adam/adam-cli/target/generated-sources/java-templates
[INFO] Source directory: /Users/heuermh/working/tmp-adam/adam-cli/target/generated-sources/java-templates added.
[INFO] 
[INFO] --- build-helper-maven-plugin:1.12:add-source (add-source) @ adam-cli_2.10 ---
[INFO] Source directory: /Users/heuermh/working/tmp-adam/adam-cli/src/main/scala added.
[INFO] 
[INFO] <<< scala-maven-plugin:3.2.2:doc (default-cli) < generate-sources @ adam-cli_2.10 <<<
[INFO] 
[INFO] 
[INFO] --- scala-maven-plugin:3.2.2:doc (default-cli) @ adam-cli_2.10 ---
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-apis_2.10/0.23.0-SNAPSHOT/adam-apis_2.10-0.23.0-20171228.152022-444.pom
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-apis_2.10/0.23.0-SNAPSHOT/adam-apis_2.10-0.23.0-20171228.152022-444.pom (4.9 kB at 36 kB/s)
[WARNING]  Expected all dependencies to require Scala version: 2.10.6
[WARNING]  com.twitter:chill_2.10:0.5.0 requires scala version: 2.10.6
[WARNING]  com.typesafe.akka:akka-remote_2.10:2.3.11 requires scala version: 2.10.6
[WARNING]  com.typesafe.akka:akka-actor_2.10:2.3.11 requires scala version: 2.10.6
[WARNING]  com.typesafe.akka:akka-slf4j_2.10:2.3.11 requires scala version: 2.10.6
[WARNING]  org.apache.spark:spark-core_2.10:1.6.3 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-jackson_2.10:3.2.10 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-core_2.10:3.2.10 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-ast_2.10:3.2.10 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-core_2.10:3.2.10 requires scala version: 2.10.0
[WARNING] Multiple versions of scala libraries detected!
[WARNING]  Expected all dependencies to require Scala version: 2.10.6
[WARNING]  com.twitter:chill_2.10:0.5.0 requires scala version: 2.10.6
[WARNING]  com.typesafe.akka:akka-remote_2.10:2.3.11 requires scala version: 2.10.6
[WARNING]  com.typesafe.akka:akka-actor_2.10:2.3.11 requires scala version: 2.10.6
[WARNING]  com.typesafe.akka:akka-slf4j_2.10:2.3.11 requires scala version: 2.10.6
[WARNING]  org.apache.spark:spark-core_2.10:1.6.3 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-jackson_2.10:3.2.10 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-core_2.10:3.2.10 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-ast_2.10:3.2.10 requires scala version: 2.10.6
[WARNING]  org.json4s:json4s-core_2.10:3.2.10 requires scala version: 2.10.0
[WARNING] Multiple versions of scala libraries detected!
/Users/heuermh/working/tmp-adam/adam-cli/src/main/scala/org/bdgenomics/adam/cli/ADAMMain.scala:87: error: not found: type About
    val about = new About()
                    ^
/Users/heuermh/working/tmp-adam/adam-cli/src/main/scala/org/bdgenomics/adam/cli/TransformAlignments.scala:500: error: not found: type About
      val about = new About()
                      ^
model contains 54 documentable templates
/Users/heuermh/working/tmp-adam/adam-cli/src/main/scala/org/bdgenomics/adam/cli/TransformAlignments.scala:347: warning: Could not find any member to link for "maybeSort".
  /**
  ^
one warning found
two errors found
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] Building ADAM_2.10: Assembly 0.23.0-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] >>> scala-maven-plugin:3.2.2:doc (default-cli) > generate-sources @ adam-assembly_2.10 >>>
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-cli_2.10/0.23.0-SNAPSHOT/maven-metadata.xml
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-cli_2.10/0.23.0-SNAPSHOT/maven-metadata.xml (1.2 kB at 11 kB/s)
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-cli_2.10/0.23.0-SNAPSHOT/adam-cli_2.10-0.23.0-20171228.152041-444.jar
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-cli_2.10/0.23.0-SNAPSHOT/adam-cli_2.10-0.23.0-20171228.152041-444.jar (234 kB at 990 kB/s)
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-versions) @ adam-assembly_2.10 ---
[INFO] 
[INFO] --- maven-enforcer-plugin:1.0:enforce (enforce-maven) @ adam-assembly_2.10 ---
[INFO] 
[INFO] --- git-commit-id-plugin:2.2.1:revision (default) @ adam-assembly_2.10 ---
[INFO] 
[INFO] --- templating-maven-plugin:1.0.0:filter-sources (filter-src) @ adam-assembly_2.10 ---
[INFO] Request to add '/Users/heuermh/working/tmp-adam/adam-assembly/src/main/java-templates' folder. Not added since it does not exist.
[INFO] 
[INFO] --- build-helper-maven-plugin:1.12:add-source (add-source) @ adam-assembly_2.10 ---
[INFO] Source directory: /Users/heuermh/working/tmp-adam/adam-assembly/src/main/scala added.
[INFO] 
[INFO] <<< scala-maven-plugin:3.2.2:doc (default-cli) < generate-sources @ adam-assembly_2.10 <<<
[INFO] 
[INFO] 
[INFO] --- scala-maven-plugin:3.2.2:doc (default-cli) @ adam-assembly_2.10 ---
Downloading from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-cli_2.10/0.23.0-SNAPSHOT/adam-cli_2.10-0.23.0-20171228.152041-444.pom
Downloaded from sonatype-nexus-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/bdgenomics/adam/adam-cli_2.10/0.23.0-SNAPSHOT/adam-cli_2.10-0.23.0-20171228.152041-444.pom (6.6 kB at 51 kB/s)
model contains 6 documentable templates
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO] 
[INFO] ADAM_2.10 .......................................... SUCCESS [  0.671 s]
[INFO] ADAM_2.10: Avro-to-Dataset codegen utils ........... SUCCESS [  3.954 s]
[INFO] ADAM_2.10: Core .................................... SUCCESS [ 33.189 s]
[INFO] ADAM_2.10: APIs for Java ........................... SUCCESS [  8.073 s]
[INFO] ADAM_2.10: CLI ..................................... SUCCESS [  7.599 s]
[INFO] ADAM_2.10: Assembly ................................ SUCCESS [  3.455 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 57.215 s
[INFO] Finished at: 2017-12-28T14:03:03-06:00
[INFO] Final Memory: 47M/640M
[INFO] ------------------------------------------------------------------------
```

--- a/scripts/publish-scaladoc.sh
+++ b/scripts/publish-scaladoc.sh
@@ -11,7 +11,7 @@ git clone -b gh-pages git@github.com:bigdatagenomics/adam.git adam-docs
 sha1=$(git log -1 --pretty=format:%H)
 
 # generate scaladoc
-mvn scala:doc
+mvn generate-sources scala:doc
 
 # generate markdown docs
 cd docs

